Formula/f/ffmpeg.rb

```diff
@@ -159,8 +160,8 @@ def install
       --disable-indev=jack
     ]
 
-    # Needs corefoundation, coremedia, corevideo
-    args += %w[--enable-videotoolbox --enable-audiotoolbox] if OS.mac?
+    # Needs corefoundation, coremedia, corevideo, and openal
+    args += %w[--enable-videotoolbox --enable-audiotoolbox --enable-openal] if OS.mac?
```

Comment:
We are fairly conservative with enabling additional features within `ffmpeg`, because the formula can quickly become bloated if many options are enabled (this introduces additional dependencies etc..)

Would the addition of `openal` be useful for a majority of `ffmpeg` users?

Noting that there is a third-party tap that maintains a version of `ffmpeg` that can have different options installed (the binary is built locally) - https://github.com/homebrew-ffmpeg/homebrew-ffmpeg

Reply:
And the third-party also set `openal` as an optional dependency...

Reply:
>And the third-party also set `openal` as an optional dependency...

… because we don’t think that the majority of the users need it.